app/src/datasets/chorus_dt/places_resolution.py

```diff
@@ -138,6 +130,22 @@ def resolve_place(places):
     return places, places_dict
 
 
+
+def filter_trips(trips: pd.DataFrame):
+
+    """
+
+    Tag trips according to rough heuristic (to be improved):
+     - all non-plane trips > 800km
+     - all TC trips
+
+    """
+    trips["fiable"] = True
+    trips.loc[(trips.index.get_level_values("prestation") != "A") & (trips["distance"] > 799), "fiable"] = False
```

Comment:
I remember this didn't work for me because "prestation" wasn't in the index. Should we try 
```suggestion
    trips.loc[(trips["prestation"] != "A") & (trips["distance"] > 799), "fiable"] = False
```

Reply:
I think the reason is that we loose the multiindexing when saving the file. In the script it works fine.

Reply:
However it won't be an index when we load it back from csv to pandas.